### ISSUE TITLE
Are ysm models in .glb format? and can they steal the 3d models?

### ISSUE THREAD

**Issue by Desmond051** [TartaricAcid/ysm#122]
He asked me if it is possible to steal or decode .ysm files because I sent him a 3d model in .ysm format. A few days later, he sent me an image where he had used a 3d model viewer, where my friend had accessed the 3d model that was in the .ysm file.

According to what he told me, he first converted the file extension .ysm to the extension .usdz

Are .ysm files 3d models in .glb format?

**Comment by KLuoNuoYa** [comment]
no, ysm file is encrypted by a unique method and have no way to use in other pplication

**Comment by KLuoNuoYa** [comment]
What version of ysm mod did you export with?

**Comment by Desmond051** [comment]
> ¿Con qué versión del mod ysm exportaste?

version for Minecraft 1.16.5

**Comment by KLuoNuoYa** [comment]
> version for Minecraft 1.16.5

this is minecraft version, not ysm version. please take a screenshot at model selection menu
